packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -237,12 +239,6 @@ export const LivenessCameraModule = (
     setIsCameraReady(true);
   };
 
-  const beginLivenessCheck = React.useCallback(() => {
-    send({
-      type: 'BEGIN',
-    });
-  }, [send]);
```

Comment:
As noted in `FaceLivenessDetectorCore`, think the `beginLivenessCheck` declaration should remain here. Additionally, we don't need `useCallback` here, `beginLivenessCheck` is just being passed to a `Button` which is not an expensive re-render, and the `Button` is only rendered when `isStartView` is `true`